### PR TITLE
Tcl script documentation fixes

### PR DESCRIPTION
**Pull request by nitz**
Found myself chasing my tail a bit trying to figure out how to run scripts. Figured I'd update the `-h` output and the manual so people after me chase their tails less! 🙂

### CODE REVIEW

File: manual/command-reference-manual.tex [hunk 1]
```diff
@@ -6141,21 +6141,27 @@ \section{synth\_xilinx -- synthesis for Xilinx FPGAs}
         write_blif 
 \end{lstlisting}
 
-\section{tcl -- execute a TCL script file}
```
Comment (whitequark) [inline manual/command-reference-manual.tex:6144]: This file is autogenerated from the help text in the C++ file, so your changes need to be done there.

File: kernel/driver.cc [hunk 2]
```diff
@@ -268,8 +268,8 @@ int main(int argc, char **argv)
 		printf("    -s scriptfile\n");
 		printf("        execute the commands in the script file\n");
 		printf("\n");
-		printf("    -c tcl_scriptfile\n");
-		printf("        execute the commands in the tcl script file (see 'help tcl' for details)\n");
+		printf("    -c script\n");
+		printf("        execute the commands in the tcl script file (see 'help script' for details)\n");
```
Comment (whitequark) [inline kernel/driver.cc:272]: Actually, this is just completely incorrect. The Yosys `script` command does not use Tcl for scripting.

Comment (nitz) [reply kernel/driver.cc:272]: At least now I feel like my confusion was legitimate, haha!

When I ran 'yosys -h', the parameter help was the one there that came out of 'driver.cc'. But when I ran 'yosys -h tcl_scriptfile' like it suggested, that wasn't even a command. The only thing I saw when running 'help' directly, was 'script', which definitely was what I wanted.

Have I just massively got my wires crossed here?

Comment (whitequark) [reply kernel/driver.cc:272]: Where does it suggest to run `yosys -h tcl_scriptfile`? I can't find it.

Comment (nitz) [reply kernel/driver.cc:272]: Okay, let me list my numerous mistakes:

I was under the impression that the tcl scripting *was* yosys scripting. I see now that they are two separate things. That clears up a lot of my misconceptions.

Second: my apologies, it was actually suggesting to run 'yosys -h tcl'. I got "no command or cell type tcl" when I ran that.

I've now got to the bottom of *that*: I'm using a nightly build from open tool forge, that is apparently built *without* tcl support, which explains why I never could find the command -- I simply didn't have it 😬

In short: I'm sorry to have wasted your time with my misunderstanding(s)! Maybe all I really should have submitted as a PR is perhaps a conditional compile out for the -c help line if tcl support isn't enabled.

Comment (whitequark) [reply kernel/driver.cc:272]: > Maybe all I really should have submitted as a PR is perhaps a conditional compile out for the -c help line if tcl support isn't enabled.

Sounds good to me!

**Review by whitequark — APPROVED** [review]

**Comment by nitz** [comment]
Thanks for going back and forth with me — apologies again for my massive misunderstanding, very much appreciate you nudging me in the right direction!

**Comment by whitequark** [comment]
No problem.
